### PR TITLE
check docker is installed when running convox start

### PR DESCRIPTION
**Pull request by awsmsrc**
## Release Playbook
- [ ] Rebase against master
- [ ] Release branch ()
- [ ] Pass CI ()
- [ ] Code review
- [ ] Merge into master
- [ ] Release master ()
- [ ] Pass CI ()
- [ ] Update staging rack
- [ ] Edit [Rack release record](https://github.com/convox/rack/releases) and/or update [docs](https://github.com/convox/convox.github.io)
- [ ] Publish release
- [ ] Release CLI

### CODE REVIEW

**Comment by awsmsrc** [comment]
screen shot 2016-05-18 at 7 39 54 pm

File: cmd/convox/start.go [hunk 1]
```diff
@@ -42,6 +44,14 @@ func cmdStart(c *cli.Context) {
 		stdcli.QOSEventSend("cli-start", distinctId, stdcli.QOSEventProperties{Error: err})
 	}
 
+	dockerTest := exec.Command("docker", "images")
+	err = dockerTest.Run()
+	if err != nil {
+		stdcli.QOSEventSend("cli-start", distinctId, ep)
```
Comment (nzoschke) [inline cmd/convox/start.go]: We don't need to track this as a start exception. It's a well-handled validation now.

Comment (awsmsrc) [inline cmd/convox/start.go]: this is not tracked as an exception, just a metric, the ep var has no error in it

Comment (nzoschke) [inline cmd/convox/start.go]: Ah, well its not a successful start either, so no tracking is required.

**Comment by ddollar** [comment]
Merging on #668

**Pull request closed without merging**